Gems/OpenXRVk/Code/Source/OpenXRVkInstance.cpp

```diff
@@ -88,26 +85,11 @@ namespace OpenXRVk
 
     AZ::RHI::ResultCode Instance::InitInstanceInternal(AZ::RHI::ValidationMode validationMode)
     {
-#ifdef AZ_PLATFORM_ANDROID
-        // Initialize the loader for Android platform
+#if OPENXRVK_NEEDS_INITIALIZE_XR_LOADER
```

Comment:
For PAL defines like this it's common to use the word `TRAIT` in the name (e.g. `OPENXRVK_TRAIT_NEEDS_INITIALIZE_XR_LOADER`).

Reply:
Excellent point. I'll change it.